### ISSUE TITLE
Squelch track slogan

### ISSUE THREAD

**Issue by jidanni** [openstreetmap/iD#4879]
Here we observe the track making software's slogan is dogging along in every view.
![de](https://user-images.githubusercontent.com/1230959/37334087-e1266b8c-26e5-11e8-9617-1b5126ec7255.jpg)
No matter where the user moves along the track, the "blasted" slogan is right there following along.

Perhaps iD could help squelch such annoyances, so the user could concentrate on the real notes he added to the track.

Once is enough. Not following us around all day please.

Seen on e.g., https://www.openstreetmap.org/edit?gpx=2585627#map=21/24.18174/120.86479 no matter how much you zoom.

**Comment by bhousel** [comment]
Yeah this could be better.. It seems that D3.js just places it wherever it thinks the centroid of the path is after being clipped.

I do want to keep the label somewhere though. Maybe we could put it up top in a dedicated place.  I'm also thinking of the idea that if we allow users to show multiple traces at the same time, it would be useful to have a legend and different colors for each one.

**Comment by jidanni** [comment]
![0](https://user-images.githubusercontent.com/1230959/38916842-6712aa48-431b-11e8-8256-94b43da59d5e.jpg)
